modules/perception/tool/export_sensor_data/export_sensor_data.cc

```diff
@@ -238,34 +241,34 @@ bool ExportSensorData::GetCarLinearSpeed(double timestamp,
     AERROR << "Param car_linear_speed NULL error.";
     return false;
   }
-  if (gps_buffer_.empty()) {
+  if (localization_buffer_.empty()) {
     AWARN << "Rosmsg buffer is empty.";
     return false;
   }
-  if (gps_buffer_.front().first - 0.1 > timestamp) {
+  if (localization_buffer_.front().first - 0.1 > timestamp) {
     AWARN << "Timestamp (" << GLOG_TIMESTAMP(timestamp)
           << ") is earlier than the oldest "
-          << "timestamp (" << gps_buffer_.front().first << ").";
+          << "timestamp (" << localization_buffer_.front().first << ").";
     return false;
   }
-  if (gps_buffer_.back().first + 0.1 < timestamp) {
+  if (localization_buffer_.back().first + 0.1 < timestamp) {
     AWARN << "Timestamp (" << GLOG_TIMESTAMP(timestamp)
           << ") is newer than the latest "
-          << "timestamp (" << gps_buffer_.back().first << ").";
+          << "timestamp (" << localization_buffer_.back().first << ").";
     return false;
   }
   // loop to find nearest
   double distance = 1e9;
-  int idx = gps_buffer_.size() - 1;
+  int idx = localization_buffer_.size() - 1;
   for (; idx >= 0; --idx) {
-    double temp_distance = fabs(timestamp - gps_buffer_[idx].first);
+    double temp_distance = fabs(timestamp - localization_buffer_[idx].first);
     if (temp_distance >= distance) {
       break;
     }
     distance = temp_distance;
   }
   const auto &velocity =
-      gps_buffer_[idx + 1].second.localization().linear_velocity();
+      localization_buffer_[idx + 1].second.pose().linear_velocity();
```

Comment:
It seems that [idx + 1] can cause a memory error ? (e.g., line 266 excuted during the first for loop)